contrib/Dockerfile.test

```diff
@@ -18,17 +18,16 @@ RUN apk add --no-cache $PACKAGES && \
   make install
 
 # Final image
-FROM alpine:edge
+FROM alpine:latest
 
 # Install ca-certificates
-RUN apk add --update ca-certificates
-WORKDIR /root
```

Comment:
do wonder if its a good idea to remove the `WORKDIR` it may be relevant to the script being executed.

Could you verify there is no reference to `$WORKDIR` please?

Reply:
```diff
- WORKDIR /root
```

In baseimage which is alpine, they specify WORKDIR so it's like we don't have to specify it.

```diff
- RUN apk add --update ca-certificates
```

Kindly keep this